cbdr/cortex
# COR-692: Refine Employer Blog Categories

**Purpose:**
Changes and refines to categories on the Employer Blog per @justin3thompson's specifications

**JIRA:**
https://cb-content-enablement.atlassian.net/browse/COR-692

**Changes:**
* Changes to setup
  * N/A

* Architectural changes
  * N/A

* Migrations
  * N/A
  
* Library changes
  * N/A

* Side effects
  * N/A

**How to Verify These Changes**
* Specific pages to visit
  * New Employer Blog Page

* Steps to take
  * Scroll Down to see Categories
  * There should now only be the three


**Relevant PRs/Dependencies:**
N/A

**Additional Information**
N/A
